distribution/docs/src/main/resources/_contents/_endpoints/application-endpoint-contents.adoc

```diff
@@ -0,0 +1,19 @@
+==== Application Upload Endpoint
+
+The Application Upload Endpoint enables uploading new and upgraded applications to the system.
+
+===== Installing Application Upload Endpoint
+
+The Application Upload Endpoint is installed by default with a standard installation as part of the ${ddf-admin} application.
+
+===== Application Upload Endpoint
+
+The Application endpoint is available at https://<${secure_url}/services/application.
```

Comment:
Looks like this renders two `https://` think you can remove the first `https://` and just use ${secure_url}